scalanativelib/worker/0.4/src/ScalaNativeWorkerImpl.scala

```diff
@@ -16,12 +18,19 @@ import scala.scalanative.build.{
   NativeConfig => ScalaNativeNativeConfig
 }
 import scala.scalanative.nir.Versions
-import mill.scalanativelib.api.{GetFrameworkResult, LTO, NativeConfig, NativeLogLevel, ReleaseMode}
-import sbt.testing.Framework
-
 import scala.scalanative.testinterface.adapter.TestAdapter
 
+import scala.util.{Success, Try}
+
 class ScalaNativeWorkerImpl extends mill.scalanativelib.api.ScalaNativeWorkerApi {
+  private def patchIsGreaterThanOrEqual(number: Int) = {
+    val patch = Versions.current.stripPrefix("0.4.")
```

Comment:
> I wonder if we could use `mill.scalalib.dependency.versions.Version` for the comparison?

We can't since here we are in a separate classpath that doesn't depend on Mill.
When Mill `0.5` is out it will be broken either way because this is the worker for `0.4` and it's used only for `0.4` version. My idea was to ship this and then evolve it when `0.5` is out (depending if the `0.5` tooling will be backwards compatible with `0.4` or not).